core/templates/pages/exploration-player-page/layout-directives/audio-bar.component.html

```diff
@@ -162,6 +162,7 @@
     flex-wrap: wrap;
     justify-content: center;
     width: 700px;
+    padding-top: 25px;
```

Comment:
Please fix the linting tests, make sure the properties are written in lexicographically sorted order.

Reply:
Sure